engine/src/flutter/shell/platform/android/io/flutter/embedding/engine/mutatorsstack/FlutterMutatorsStack.java

```diff
@@ -47,6 +47,7 @@ public class FlutterMutator {
     @Nullable private Rect rect;
     @Nullable private Path path;
     @Nullable private float[] radiis;
+    @Nullable private float opacity;
```

Comment:
Somewhere in this pr can you javadoc what the expected range of values of opacity can be and what you do if the values are invalid (like negative)